File: src/main/kotlin/com/itmo/microservices/demo/order/impl/dao/OrderItemRepository.java
package com.itmo.microservices.demo.order.impl.dao;

import com.itmo.microservices.demo.order.impl.entity.OrderItem;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface OrderItemRepository extends JpaRepository<OrderItem, UUID> {
}

File: src/main/kotlin/com/itmo/microservices/demo/lib/common/order/repository/OrderItemRepository.kt
package com.itmo.microservices.demo.lib.common.order.repository

import com.itmo.microservices.demo.lib.common.order.entity.OrderEntity
import com.itmo.microservices.demo.lib.common.order.entity.OrderItemEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface OrderItemRepository : JpaRepository<OrderItemEntity, UUID>{
    fun findByOrderEntity(orderEntity: OrderEntity): List<OrderItemEntity>?
}
